parm/config/config.resources

```diff
@@ -536,20 +552,20 @@ elif [[ "${step}" = "fcst" || "${step}" = "efcs" ]]; then
     unset _CDUMP _CDUMP_LIST
     unset NTASKS_TOT
 
-elif [ ${step} = "ocnpost" ]; then
+elif [[ ${step} = "ocnpost" ]]; then
 
     export wtime_ocnpost="00:30:00"
     export npe_ocnpost=1
     export npe_node_ocnpost=1
     export nth_ocnpost=1
-    export memory_ocnpost="96G"
+    export memory_ocnpost="88G"
```

Comment:
So Jet has 96GB/node but it caps it at 88GB? The alternative would be to run ocnpost on two nodes on Jet instead of adjusting memory for everywhere?

Reply:
That's correct, Jet caps requests at 88GB, or at least less than 89GB.  I'm happy to increase the number of nodes for Jet if desired.

Reply:
88GB may work on the other platforms but we'd need to run tests on each to be sure. Given Jet has this cap that the other platforms don't, I'm inclined to have you add a check for Jet here and increase the job to use two nodes (setting `npe_ocnpost=2` on Jet should do it). Perhaps add a comment noting the cap and it can be reexamined later. I think ocnpost still doesn't work on WCOSS2 anyway so work is needed for this job.

@WalterKolczynski-NOAA any objections/comments on the resources for this job?

Reply:
This configuration is giving me an error that the memory specification cannot be satisfied.  This sets the `sbatch` parameter `--mem=98304`.  The documentation notes that `--mem` specifies the memory required per node.  So I think I need to specify a memory request of `48GB` for Jet over two nodes.  I'll make this change and test it.

Reply:
@KateFriedman-NOAA This worked.  The job reported a max memory usage of ~7GB.  However, I'm not sure whether memory can be shared across nodes for this job since it is serial, so I'm not sure if this will be adequate if there is a case that gets to be above the 48GB mark.  Thoughts?

Reply:
Noted. For now we're not running coupled mode higher than C384 so we'll cross the memory bridge another day if needed. Thanks for making the needed adjustment to run the job with 2 nodes on Jet.